### PR TITLE
Add timestmp->timestamp and its variations

### PR DESCRIPTION
**Pull request by fkmy**
43.5k files:
https://github.com/search?q=timestmp&type=code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -56938,6 +56938,7 @@ timestemp->timestamp
 timestemps->timestamps
 timestmap->timestamp
 timestmaps->timestamps
+timestmp->timestamp
```
Comment (skangas) [inline codespell_lib/data/dictionary.txt:56941]: How about adding `timestmps->timestamps` too?

Comment (skangas) [reply codespell_lib/data/dictionary.txt:56941]: Perhaps also variation timestamped, timestamping.

Comment (fkmy) [reply codespell_lib/data/dictionary.txt:56941]: Appreciate it, @skangas !
I have added the variations you suggested.

**Review by skangas — APPROVED** [review]

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @fkmy and @skangas.

**Pull request merged**